gdpr/utils.py

```diff
@@ -19,4 +21,18 @@ def delete_user_data(user: User, dry_run: bool) -> None:
     if dry_run:
         raise DryRunException("Dry run. Rollback delete transaction.")
     else:
+        user_id = user.id
         user.delete()
+        content_type = ContentType.objects.get_for_model(User)
+        # Create Augitlog entry for deletion of user instance. Store only
+        # the object id, ensuring we do not store the personal data that was deleted.
```

Comment:
I've understood that an object ID to a user object is still Personal Identifiable Information. But if the user object doesn't exist anymore in the DB, I guess it doesn't identify the person anymore.

Reply:
That could probably be the case, but I'm thinkin it is fine since we delete the user. I can't think of anything else from the user to store in the auditlog than the id. If we don't have any information to store, then the auditlog doesn't make sense.

Reply:
Agreed